.gitattributes

```
@@ -5,6 +5,9 @@
 # encrypted files/folders
 env/** filter=git-crypt diff=git-crypt
 .env filter=git-crypt diff=git-crypt
+.env.last filter=git-crypt diff=git-crypt
+.env.atb filter=git-crypt diff=git-crypt
+.env.nfk filter=git-crypt diff=git-crypt
```

Dette kan sikkert forenkles til:
```suggestion
.env.* filter=git-crypt diff=git-crypt
```
?